### PR TITLE
First pull request.

### PR DESCRIPTION
**Pull request by benwmx**
first, pull request to the ` "Hello Microverse" ` project.

### CODE REVIEW

**Comment by mikenath223** [comment]
Hi , 

You made a great effort in this project, but unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that will be invalidated for this reason:
-  Please fix all linting issues before requesting a new review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding. Cheers!🚀🚀

If something isn't 100% clear, you can leave comments or questions in this PR thread.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **don't count** into the code reviews limit._

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @benwmx, 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- Add linter files to the project.


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (mikenath223) [inline .github/workflows/linters.yml:49]: - [ ] As stated in the guidelines [here](https://github.com/microverseinc/linters-config/tree/master/html-css), please add both the Webhint and Stylelint config files.

**Review by samroodali — APPROVED** [review]
Hi @benwmx , 

# Your project is approved :partying_face: 
it's time to merge it :shipit: 

Your project is complete!  Thank you for adding the linter files for GitHub actions. 
Linters are really helpful. Here are some of the reasons:
1. They format your code automatically, you no longer have to worry about it.
2. Improves collaborations on your project as everyone can have the same code linting rules
3. It will be better for code reviewers as it reduces the code difference that shows up on Git.

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**